frontend/src/app/apps/page.tsx

```diff
@@ -23,6 +23,7 @@ export default function App() {
 		try {
 			const res = await fetch(`/api/applications`, {
 				cache: 'no-cache',
+				credentials: 'include',
```

Comment:
Not sure why we need this, we have cookies that we pull from the .ts file in the api.

Reply:
We don't. That was from trying to debug not getting cookies in the backend, didn't work obviously so I'll remove any occurrence of `credentials: 'include'`